app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/workers/RefreshAgendaWeekWorker.kt

```diff
@@ -116,32 +140,32 @@ class RefreshAgendaWeekWorker @AssistedInject constructor(
     }
 }
 
-// • Start the one-time Refresh 'Agenda Week' Worker
-fun startRefreshAgendaWeekWorker(applicationContext: Context) {
-    val refreshAgendaWeekConstraints: Constraints = Constraints.Builder().apply {
-        setRequiredNetworkType(NetworkType.CONNECTED)
-    }.build()
-    val data = Data.Builder()
-        .putString(
-            RefreshAgendaWeekWorker.PARAMETER_START_DATE,
-            ZonedDateTime.now().truncatedTo(ChronoUnit.DAYS).toString()
-        )
-        .build()
-    val name = RefreshAgendaWeekWorker.WORKER_NAME
-    val agendaWeekWorkRequest =
-        OneTimeWorkRequestBuilder<RefreshAgendaWeekWorker>()
-            .setConstraints(refreshAgendaWeekConstraints)
-            .setInputData(data)
-            .addTag(name)
-            .addTag("For 10 days around ${data.getString(RefreshAgendaWeekWorker.PARAMETER_START_DATE)}")
-            .addTag(TASKY_WORKERS_TAG)
-            .setExpedited(OutOfQuotaPolicy.RUN_AS_NON_EXPEDITED_WORK_REQUEST)
-            .setBackoffCriteria(BackoffPolicy.LINEAR, 1, TimeUnit.MINUTES)
-            .build()
-    WorkManager.getInstance(applicationContext)
-        .enqueueUniqueWork(
-            name,
-            ExistingWorkPolicy.REPLACE,
-            agendaWeekWorkRequest
-        )
-}
\ No newline at end of file
+//// • Start the one-time Refresh 'Agenda Week' Worker
+//fun startRefreshAgendaWeekWorker(applicationContext: Context) {
+//    val refreshAgendaWeekConstraints: Constraints = Constraints.Builder().apply {
```

Comment:
If the app works, the comments can be removed

Reply:
will do